docs/conventions/docs/rst.rst

```diff
@@ -15,10 +15,10 @@ ReStructuredText
 General
 =======
 
-With the exception of pre-formatted text (e.g., literals and code
-blocks), lines should not exceed 72 characters. At this width, GitHub
-pull request diffs will never be soft wrapped, which aids readability
-and collaboration.
```

Comment:
Why remove the blurb about GitHub?

Reply:
See the [point above](https://github.com/crate/crate-docs-utils/pull/9#discussion_r431904942). It's anyway not important, I think, since we can just have a line standard to create uniform presentation.

Reply:
Ahhhh ooops missed that.